GitCommands/EnvironmentPathsProvider.cs

```diff
@@ -72,22 +72,22 @@ public IEnumerable<string> GetEnvironmentValidPaths()
         // TODO: optimise?
         internal static bool IsValidPath(string path)
         {
-            FileInfo fi = null;
             try
             {
-                fi = new FileInfo(path);
+                var _ = new FileInfo(path).Attributes;
```

Comment:
```suggestion
                _ = new FileInfo(path).Attributes;
```

Reply:
The change here will probably cause us grief going forward, the IO API have been changed in .NET Core 2.1...